tests/test-bad-http-host-rule-1/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 7.0.0
+
+  features:
+    - HAVE_LIBJANSSON
```

Comment:
By the way, `HAVE_LIBJANSSON` is implied by min version 7 ;-)

Reply:
Okay, will update PR